chia/farmer/farmer.py

```diff
@@ -500,7 +499,7 @@ async def update_pool_farmer_info() -> Tuple[Optional[GetFarmerResponse], Option
                         farmer_info, error_code = await update_pool_farmer_info()
                         if error_code == PoolErrorCode.FARMER_NOT_KNOWN:
                             # Make the farmer known on the pool with a POST /farmer
-                            owner_sk_and_index: Optional[PrivateKey, uint32] = find_owner_sk(
+                            owner_sk_and_index: Optional[Tuple[G1Element, uint32]] = find_owner_sk(
```

Comment:
Seems odd to me that `find_owner_sk` is returning a `G1Element` instead of a `PrivateKey`. I would fix the return type of `find_owner_sk` instead of changing the type of `owner_sk_and_index`.

Reply:
I wasn't sure which direction to go here - I also found it odd, but that is long-standing code, so I wasn't sure.

Reply:
Fixed, turns out all callers were expecting find_owner_sk to return PrivateKey - so fixed that